Editor/MobiledgeXEditorWindow.cs

```diff
@@ -108,18 +109,21 @@ public static void ImportEdgeMultiplayExample()
         {
             string sdkPath = Path.GetFullPath("Packages/com.mobiledgex.sdk");
             AssetDatabase.ImportPackage(Path.Combine(sdkPath, "Resources/Examples/EdgeMultiplay.unitypackage"), true);
+            Enhancement.EdgeMultiplayImported();
         }
 
         [MenuItem("MobiledgeX/Examples/Computer Vision", false, 20)]
         public static void ImportComputerVisionExample()
         {
             string sdkPath = Path.GetFullPath("Packages/com.mobiledgex.sdk");
             AssetDatabase.ImportPackage(Path.Combine(sdkPath, "Resources/Examples/ComputerVision.unitypackage"), true);
+            Enhancement.CVImported();
         }
 
         [MenuItem("MobiledgeX/Remove MobiledgeX", false, 40)]
         public static void RemoveMobiledgeX()
         {
+             Enhancement.SDKRemoved();
```

Comment:
Should have RemoveRequested, or moved somewhere where it actually is removed.

Reply:
I moved it inside the if statement, which will run if the user selected "Remove"